src/plugins/intel_cpu/src/nodes/mvn.cpp

```diff
@@ -1905,6 +1905,8 @@ void MVN::initSupportedPrimitiveDescriptors() {
         canUseAclExecutor = !supportedPrimitiveDescriptors.empty();
         if (canUseAclExecutor)
             return;
+        else
+            outputPrecision = Precision::FP32;
```

Comment:
1. What about inputPrecision?
2. Lets add comment which explains why precision is changes here.

Reply:
fixed